deploy/examples/datasources/Postgres.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: example-postgres-datasource
+spec:
+  name: example-postgres
+  datasources:
+  - name: Postgres
+    type: postgres
+    database: dvdrental # name of your database
+    editable: false
+    jsonData:
+      connMaxLifetime: 14400
+      maxIdleConns: 2
+      maxOpenConns: 0
+      postgresVersion: 1400
+      sslmode: disable
+      timescaledb: false
+    secureJsonData:
+      password: admin123 # password of your user
+    url: postgres.default.svc.cluster.local:5432 #change to url of your postgres instance
+    user: postgresadmin # user to access your database
```

Comment:
Silly thing but I know I will think about it in the future. Trying to add \n at the end of all our yaml files.
It's on my todo to add a CI rule for this.
Other then that I think it looks great

Reply:
Will do.
On further notice: Would it be a possibility to add an .editorconfig file to the repo?
It would be insured, that something like this might not happen as easily again.

Reply:
There have been talk about editorconfig before https://github.com/grafana-operator/grafana-operator/pull/516 if you want to take a look at it again please do.

Reply:
@NissesSenap I force-pushed the change, can you approve?